packages/examples/packages/bip32/snap.config.ts

```diff
@@ -7,12 +7,11 @@ const config: SnapConfig = {
   server: {
     port: 8001,
   },
-  polyfills: {
-    stream: true,
-  },
   stats: {
-    builtIns: false,
     buffer: false,
+    builtIns: {
+      ignore: ['crypto'],
```

Comment:
Why do we need to ignore crypto?

Is it erroneously brought into the bundles?

Reply:
There's a function for generating random numbers which checks if `crypto` exists and falls back to the native browser API otherwise.